setup.cfg

```diff
@@ -20,3 +20,5 @@ known_first_party = dramatiq
 multi_line_output = 5
 order_by_type = true
 profile = black
+# Remove annotations import once we only support python 3.14+
+add_imports = from __future__ import annotations
```

Comment:
Oh this injected the import into every file, huh?

Reply:
Yes when you run isort, this means the import will be added automatically.

So rather than needing to think about which files need it or use annotations, its automatically used everywhere, sort of like black formatting :)

Reply:
I’m surprised neither flake8 nor pylint notice it as an unused import. But I guess [__future__](https://docs.python.org/3/library/__future__.html) isn’t your standard imported package…

Reply:
Yes they're really flags to the compiler